packages/subscription-controller/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.0]
+
 ### Added
 
 - Initial release of the subscription controller ([#6233](https://github.com/MetaMask/core/pull/6233))
```

Comment:
Nit: Usually for the initial release, we just have an "Initial release" change entry, and we delete the rest (since they haven't really changed). Whatever you'd prefer is fine though, it doesn't strongly matter.